static/src/javascripts/projects/common/modules/analytics/google.js

```diff
@@ -129,7 +130,7 @@ const trackPerformance = (
             sendPerformanceEvent(event);
         } else {
             mediator.on('modules:ga:ready', sendDeferredEventQueue);
-            config.get('googleAnalytics.timingEvents').push(event);
+            timingEvents.push(event);
```

Comment:
Outside the scope of this PR since this also applies to the code before the change... 


However, I am confused as to the intent of this code 🤔 

IIRC, `config.get()` does not let the config object be mutated, so what is the purpose of `.push(event)`? Surely the `timingEvents` queue is not accessible anywhere else?